mountinfo/mountinfo_freebsdlike.go

```diff
@@ -6,9 +6,9 @@ package mountinfo
 import "golang.org/x/sys/unix"
 
 func getMountinfo(entry *unix.Statfs_t) *Info {
-	var mountinfo Info
-	mountinfo.Mountpoint = unix.ByteSliceToString(entry.Mntonname[:])
-	mountinfo.FSType = unix.ByteSliceToString(entry.Fstypename[:])
-	mountinfo.Source = unix.ByteSliceToString(entry.Mntfromname[:])
-	return &mountinfo
+     return &Info{
```

Comment:
Linter is complaining about this part; I haven't checked locally, but possibly indentation is wrong in this file?

```
mountinfo_freebsdlike.go:9: File is not `gofumpt`-ed (gofumpt)
[18](https://github.com/moby/sys/runs/6746317554?check_suite_focus=true#step:4:19)
     return &Info{

Reply:
Oh, absolutely. Sorry for the hustle.